src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -13,6 +13,7 @@
     <IsOSXLike Condition="'$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</IsOSXLike>
     <SupportsArmIntrinsics Condition="'$(Platform)' == 'arm64'">true</SupportsArmIntrinsics>
     <SupportsX86Intrinsics Condition="'$(Platform)' == 'x64' or ('$(Platform)' == 'x86' and '$(TargetsUnix)' != 'true')">true</SupportsX86Intrinsics>
+    <SupportsX86Intrinsics Condition="'$(TargetsBrowser)' == 'true'">false</SupportsX86Intrinsics>
```

Comment:
What is `Platform` set to for Browser/Wasm? Should the fix be to correct what the platform is set to for Wasm?

Reply:
Platform is 'wasm' i think.

Reply:
It means that none of the conditions above should be kicking in. Why does `SupportsX86Intrinsics` need to be set explicitly to false for Wasm then?

Reply:
you're right, just checked both x86 and Arm APIs, all of them throw PNSE as expected, sorry